CustomCommands.c

```
@@ -0,0 +1,51 @@
+static void cmd_pkg_ls(ScreenManager* scr) {
+    show_temp_message("pkg-ls - Memory Hog Package", 
+                      "Package finding feature\n\n"
+                      "To implement: find highest memory process\n"
+                      "and query dpkg/rpm for package name");
+}
+
+static void cmd_max_ls(ScreenManager* scr) {
+    show_temp_message("max-ls - Top Memory Processes", 
+                      "Top memory processes feature\n\n"
+                      "To implement: sort by memory and show top N");
+}
```

Cppcheck (2.20.0)

[style] The function 'ListItem_getRef' is never used.

(unusedFunction)